package.json

```diff
@@ -24,10 +24,16 @@
     "react-component"
   ],
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test",
-    "lint": "eslint ./src/**/*.ts"
+    "build": "npm run build-esm && npm run build-node && npm run build-browser && npm run build-browser-min && npm run build-types",
+    "build-esm": "esbuild src/index.ts --bundle --outfile=dist/react-oidc-context.mjs --format=esm",
+    "build-node": "esbuild src/index.ts --bundle --outfile=dist/react-oidc-context.cjs --platform=node",
+    "build-browser": "esbuild src/index.ts --bundle --outfile=dist/react-oidc-context.js --platform=browser",
+    "build-browser-min": "npm run build-browser -- --minify --outfile=dist/react-oidc-context.min.js",
```

Comment:
I think we can omit these targets, they're only useful to people loading library code from a CDN and are writing React code without JSX.

Otherwise we need to pick a good global variable with `--global-name=<...>` so users can do something like

```js
React.createElement(ReactOidcContext.AuthProvider, ...)
```